samples/Nancy.Demo.Hosting.AspnetCore/Startup.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Builder;

```

Comment:
Can we move the using statements inside the namespace, to align with the rest of the Nancy code